src/functionaltest/resources/functional-test.properties

```diff
@@ -1,4 +1,6 @@
 test.selenium.firefox.BZip2File.url.linux = https://ftp.mozilla.org/pub/firefox/releases/61.0.1/linux-x86_64/en-GB/firefox-61.0.1.tar.bz2
-ei.backend.instances.list.json.content=[{"name":"EI-Backend-1","host":"localhost","port":8070,"contextPath":"","https":false,"defaultBackend":true}, {"name":"EI-Backend-2","host":"localhost","port":8077,"contextPath":"","https":false,"defaultBackend":false}, {"name":"EI-Backend-3","host":"localhost","port":8078,"contextPath":"","https":false,"defaultBackend":true}]
+ei.backend.instances.list.json.content=[{"name":"EI-Backend-1","host":"localhost","port":8070,"contextPath":"","https":false,"defaultBackend":true}, \
```

Comment:
A lot of white spaces here,, or does just looks strange in the review view.

Reply:
I've indented this like the EI Documentation url below. It might look a bit wierd in the github code review

Reply:
ok